### PR TITLE
Search, badges, email fix

### PR DESCRIPTION
**Pull request by andrewhalle**
- simple search available for logged in user, searches by name of app, org, user
- pivotal tracker link, travis badge, cc badges
- fix emailer to send proper link out
- add repo url to engagements

### CODE REVIEW

File: app/controllers/iterations_controller.rb [hunk 1]
```diff
@@ -53,7 +53,8 @@ def get_customer_feedback
         pf.iteration = iter
         pf.save
 
-        url = "http://localhost:3000/feedback/#{eng.id}/#{iter.id}"
+        host = request.env["REQUEST_URI"].split("/")[2]
```
Comment (armandofox) [inline app/controllers/iterations_controller.rb]: why not `request.host` or `request.host_with_port` ?
http://apidock.com/rails/ActionDispatch/Http/URL/host_with_port

File: app/controllers/search_controller.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class SearchController < ApplicationController
```
Comment (armandofox) [inline app/controllers/search_controller.rb]: can we standardize on 2 spaces, no use of tabs?

**Comment by andrewhalle** [comment]
converted to 2 spaces

**Comment by andrewhalle** [comment]
switched to request.host_with_port

**Pull request merged**